common/evm_block_ref.go

```diff
@@ -7,6 +7,11 @@ import (
 	"strings"
 )
 
+// List of methods that static result is expected for. i.e. result will not change over time
+var EvmStaticMethods = map[string]bool{
+	"eth_chainId": true,
```

Comment:
how about these methods?

`net_version`:  Returns the current network ID
`web3_clientVersion`:  Returns the current client version.
`eth_protocolVersion`: Returns the current Ethereum protocol version.

Reply:
I thought about adding these as well, the biggest question is "if it correct to cache data from 'one of' upstreams" as you might have multiple upstreams each different values for some of these methods?

Reply:
hmm that is indeed an interesting problem. they are static in the context of 1 upstream node, but here it could be different for different nodes.